pycbc/distributions/sky_location.py

```diff
@@ -31,5 +33,47 @@ class UniformSky(angular.UniformSolidAngle):
     _default_polar_angle = 'dec'
     _default_azimuthal_angle = 'ra'
 
+class FisherDist():
+    """A distribution that returns a random (ra, dec) angle drawn from the
+    Fisher distribution. Assume that the concentration parameter (kappa)
+    is large so that we can use a Rayleigh distribution about the north
+    pole and rotate it to be centered at the (ra, dec) coordinate mu.
```

Comment:
If may be worth checking if the assumption on `kappa` is being violated, and raising a warning or maybe even a ValueError.

Reply:
Theoretically it says in the limit kappa --> infinity . So, is it okay if I mention it in the docstring so that it will be understood that the class works for only high values of kappa ?

Reply:
The suggested change has been made